### PR TITLE
Bnb/spatial dc

### PR DESCRIPTION
**Pull request by bnb32**

### CODE REVIEW

File: sup3r/models/base.py [hunk 1]
```diff
@@ -717,12 +729,74 @@ def meta(self):
...
+        ----------
+        s_enhance : int
+            Spatial enhancement factor
+        t_enhance : int
+            Temporal enhancement factor
+        smoothing : float
+            Smoothing parameter used in gaussian filtering of coarsened data
+        """
+        if self.s_enhance is None:
+            self.meta['s_enhance'] = s_enhance
```
Comment (grantbuster) [inline sup3r/models/base.py:764]: You could make these three if/elif statements into a for loop. Maybe even have the input be kwargs so you could iterate easily.

File: sup3r/models/base.py [hunk 2]
```diff
@@ -717,12 +729,74 @@ def meta(self):
...
+        Parameters
+        ----------
+        s_enhance : int
+            Spatial enhancement factor
+        t_enhance : int
+            Temporal enhancement factor
+        smoothing : float
+            Smoothing parameter used in gaussian filtering of coarsened data
+        """
+        if self.s_enhance is None:
```
Comment (grantbuster) [inline sup3r/models/base.py:763]: You could make these three if/elif statements into a for loop. Maybe even have the input be kwargs so you could iterate easily.

**Review by grantbuster — CHANGES_REQUESTED** [review]
Looks good!

File: sup3r/models/base.py (hunk 2 above)
Comment (bnb32) [reply sup3r/models/base.py:763]: yeah good point

**Review by grantbuster — APPROVED** [review]

**Pull request merged**